noxfiles/docker_nox.py

```diff
@@ -184,25 +188,28 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
     session.run(*build_command, external=True)
 
 
-def push_fides_images(session: nox.Session, tag_suffixes: List[str]) -> None:
-    # Publish Fides
+def session_runner(command: Tuple[str, ...], session: nox.Session) -> None:
```

Comment:
i don't _think_ this is being used? perhaps this was part of a previous iteration and was left in by accident? i think the `partial_run` takes its place if i'm not mistaken

Reply:
removed in https://github.com/ethyca/fides/pull/3492/commits/efe2d0f8b163f34ad237d170d79e26248f8d3eb0 - let me know if i was mistaken, and we can easily add back

Reply:
correct! That was before I added the `partial` solution!